src/test/java/seedu/cakecollate/model/orderitem/TypeTest.java

```diff
@@ -27,18 +27,13 @@ public void isValid() {
 
         // invalid types
         assertFalse(Type.isValidType("")); // empty string
-        assertFalse(Type.isValidType(" ")); // spaces only
-        assertFalse(Type.isValidType("123")); // numbers only
-        assertFalse(Type.isValidType("#abc$")); // containing some special characters
-        assertFalse(Type.isValidType("strawberry cake ")); //ending with a space
         assertFalse(Type.isValidType(" strawberry cake")); //starting with a space
-        assertFalse(Type.isValidType(" strawberry cake ")); //starting and ending with a space
 
         // valid types
         assertTrue(Type.isValidType("Strawberry cake")); //typical order type
         assertTrue(Type.isValidType("s")); // one character
         assertTrue(Type
-                .isValidType("Large black forest cake with "
+                .isValidType("8 \" large black forest cake with "
                         + "buttercream frosting and chocolate drizzle topping")); // long order type
     }
```

Comment:
(For your future testing purposes) I wonder if a single " will be accepted? Can be added to tests if you think it's necessary :)